components/HistoryTable.qml

```diff
@@ -207,6 +225,21 @@ ListView {
                 text: paymentId
 
             }
+            // Address book lookup
+            TextEdit {
+                readOnly: true
+                selectByMouse: true
+                id: addressBookLookupValue
+                width: 136
+                anchors.bottom: parent.bottom
+                //elide: Text.ElideRight
+                font.family: "Arial"
+                font.pixelSize:13
+                font.letterSpacing: -1
+                color: "#545454"
+                text: "(" + lookupPaymentID(paymentId) + ")"
```

Comment:
It's incoming only. It might trigger on outgoing if there's a payment id collision, though.